### DISCUSSION TITLE
@next/bundle-analyzer doesn't output HTML reports

### DISCUSSION THREAD

**Discussion by geosigno** [vercel/next.js#14759] [Help]
I'm trying to get the HTML report outputs from @next/bundle-analyzer but there is nothing. I can see the recap on the terminal with size of each page/chunk, but no visual detail report?

Any clue why? (It was working at some point)

**Comment by noctufaber** [comment]
Me too.

**Reply by geosigno** [reply]
My issue was that I had 2 different export in my next.config.js. So it was taking only the last one (or the first one).

**Reply by cadabra-ymak** [reply]
> My issue was that I had 2 different export in my next.config.js. So it was taking only the last one (or the first one).

thahks, it helps
